### PR TITLE
Some small fixes

### PR DESCRIPTION
**Pull request by bluerthanever**
For details please refer to each commits.
Main changes:
1. Fixed progress bar print
2. If no configuration item is changed, `writeConfig` will do nothing
3. Made configuration load from file before processing every fanbox artist id

### CODE REVIEW

File: PixivUtil2.py [hunk 1]
```diff
@@ -1272,10 +1276,10 @@ def setup_option_parser():
 10 - Download by Tag and Member Id
 11 - Download images from Member Bookmark
 12 - Download images by Group Id
-f1 - Download from supported artists (FANBOX)
+f1 - Download from supporting artists (FANBOX)
 f2 - Download by artist/creator id (FANBOX)
 f3 - Download by post id (FANBOX)
-f4 - Download from followed artists (FANBOX)
```
Comment (Nandaka) [inline PixivUtil2.py:1278]: grammar wise should be in passive form, right? not sure if due to british english/american english grammar difference...

Comment (bluerthanever) [reply PixivUtil2.py:1278]: i don't know anymore... i am confused by it now.
either way seems a little weird
if you think it should be passive form, please change it for me..
thanks

Comment (bluerthanever) [reply PixivUtil2.py:1278]: since the request is closed already

**Comment by ARandomScientist** [comment]
In order to download anything, I had to checkout back to `e550cee0e0ce6ed72444b342db1b6800ef361efc`.

When I launched `./PixivUtil2.py` to initialize the `config.ini` file (first run), it seemed to bring up a screenshot menu. Had to Ctrl-C several times to get out. These are the errors I received on exit:

`^[^[^C^C^C^C^C^C./PixivUtil2.py: line 14: from: command not found
./PixivUtil2.py: line 16: from: command not found
^C^C^C^C^C^C^C^C^C^C^C./PixivUtil2.py: line 29: from: command not found
./PixivUtil2.py: line 30: from: command not found
./PixivUtil2.py: line 31: from: command not found
./PixivUtil2.py: line 32: from: command not found
./PixivUtil2.py: line 33: from: command not found
./PixivUtil2.py: line 34: from: command not found
./PixivUtil2.py: line 36: DEBUG_SKIP_PROCESS_IMAGE: command not found
./PixivUtil2.py: line 37: DEBUG_SKIP_DOWNLOAD_IMAGE: command not found
./PixivUtil2.py: line 41: syntax error near unexpected token `('
./PixivUtil2.py: line 41: `    def win_getpass_with_mask(prompt='Password: ', stream=None):'`

Again, reverting to previous version resumed normal, expected behavior.

OS: Linux 5.7.6-arch1-1 #1 SMP PREEMPT x86_64 GNU/Linux

**Comment by bluerthanever** [comment]
Sorry for that... I will fix it now....

**Comment by bluerthanever** [comment]
Sorry, have to ask, what's this "screenshot menu"? I am using windows 10 and I don't really get that.
Also, is the problem "no config.ini generated at first run"?

**Comment by bluerthanever** [comment]
Sorry for the inconvenience. I just pulled a new request to roll back PixivConfig to previous version. I think it's the best.
Let me know if anything else.
Again, really sorry.

**Comment by Nandaka** [comment]
no problemo 😃

**Pull request merged**